syne_tune/blackbox_repository/simulated_tabular_backend.py

```diff
@@ -292,6 +293,9 @@ def __init__(
         self._surrogate_kwargs = (
             surrogate_kwargs if surrogate_kwargs is not None else dict()
         )
+        self._add_surrogate_kwargs = (
```

Comment:
handy shortcut: `add_surrogate_kwargs or dict()`

Reply:
This seems like an anti-pattern, when I look at stackoverflow comments

Reply:
you are right, thanks for sharing!